roles/cloud_connector/tasks/delete_source.yaml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
This new file handles fetching a source record and issuing a delete to help cleanup the receptor end to end.